PriceCalculatorKata/SpecialDiscount.cs

```diff
@@ -0,0 +1,22 @@
+﻿using PriceCalculatorKata.Common;
+
+namespace PriceCalculatorKata;
+
+public struct SpecialDiscount
```

Comment:
> why a struct?

Must've been a misclick when I created the "struct" using the context menu. Will make it a class, since it doesn't represent a single value, and could be changed in the future.